code/modules/games/tarot.dm

```diff
@@ -0,0 +1,25 @@
+//These cards certainly won't tell the future, but you can play some nice games with them.
+/obj/item/toy/cards/deck/tarot
+	name = "tarot game deck"
+	desc = "A full 78 card game deck of tarot cards. Complete with 4 suites of 14 cards, and a full suite of trump cards."
+	icon = 'icons/obj/toy.dmi'
+	icon_state = "deck_tarot_full"
+	deckstyle = "tarot"
+
+/obj/item/toy/cards/deck/tarot/populate_deck()
+	for(var/suit in list("Hearts", "Pikes", "Clovers", "Tiles"))
+		for(var/i in 1 to 10)
+			cards += "[i] of [suit]"
+		for(var/person in list("Valet", "Chevalier", "Dame", "Roi"))
+			cards += "[person] of [suit]"
+	for(var/trump in list("The Magician", "The High Priestess", "The Empress", "The Emperor", "The Hierophant", "The Lover", "The Chariot", "Justice", "The Hermit", "The Wheel of Fortune", "Strength", "The Hanged Man", "Death", "Temperance", "The Devil", "The Tower", "The Star", "The Moon", "The Sun", "Judgement", "The World", "The Fool"))
+		cards += "[trump]"
+
+/obj/item/toy/cards/deck/tarot/draw_card(mob/user)
+	. = ..()
+	var/obj/item/toy/cards/singlecard/C = .
+	var/matrix/M = matrix()
+	M.Turn(180)
```

Comment:
Move the matrix creation and rotation to inside of the if statement.  No sense in creating and manipulating a matrix if it won't be used.